examples/tictoc/js/rocky-extended.js

```diff
@@ -1,73 +1,79 @@
-if (typeof(Rocky) == "undefined") {
-    Rocky = {};
+/*eslint "no-unused-vars": [2, { "args": "none", "vars": "local" }]*/
```

Comment:
I'm not sure this is the right thing to do here. It complains about TimerService and Window being unused